src/gbastd/types/time.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TIME_H
+#define TIME_H
+
+#include "u.h"
+#include "../../defines.h"
+
+typedef struct {
+    u8 seconds;
+    u8 minutes;
+    u8 hours;
+
+    u64 tick;
```

Comment:
Time should contain only processed time, **Hours, minutes, seconds**. Tick should only be member of **Timer** struct.
![image](https://user-images.githubusercontent.com/36482401/191184892-9797d6f8-bf6b-4228-9bbf-757f91e265a8.png)